ksql-examples/src/main/java/io/confluent/ksql/datagen/DelimitedConsumer.java

```diff
@@ -45,8 +45,8 @@
   private static Serde<GenericRow> getGenericRowSerde() {
     if (genericRowSerde == null) {
       Map<String, Object> serdeProps = new HashMap<>();
-
-      final Serializer<GenericRow> genericRowSerializer = new KsqlDelimitedSerializer();
+      // Set the schema when using this class.
```

Comment:
what info is this comment adding? `null` is being passed in as the schema

Reply:
Actually, `DelimitedConsumer.java` and `JsonConsumer.java` are not used anymore. Removed both of them.